### PR TITLE
chore: remove unnecessary directories

### PR DESCRIPTION
**Pull request by suztomo**

### CODE REVIEW

**Comment by suztomo** [comment]
ci/build (11) failed. It's bazel that used the tests directory.

```
/home/runner/.cache/bazel/_bazel_runner/1f815583ebc8da6875b9074f31ba9a5d/external/bazel_tools/tools/build_defs/repo/http.bzl:355:31: in <toplevel>
(21:44:45) INFO: Analyzed 0 targets (1 packages loaded, 0 targets configured).
(21:44:45) INFO: Found 0 test targets...
(21:44:45) INFO: Elapsed time: 6.808s, Critical Path: 0.02s
(21:44:45) INFO: 1 process: 1 internal.
(21:44:45) ERROR: No test targets were found, yet testing was requested
Error: Process completed with exit code 4.
```

File: gapic-generator-java/scripts/BUILD.bazel [hunk 1]
```diff
@@ -1,3 +0,0 @@
-package(default_visibility = ["//test/integration:__pkg__"])
```
Comment (blakeli0) [inline gapic-generator-java/scripts/BUILD.bazel:1]: We may still need this folder for updating goldens. But as long as all the bazel tests are passing, we should be good.

**Comment by suztomo** [comment]
```
bazel --batch test //:units --noshow_progress --test_output=errors
...
(21:48:08) ERROR: No test targets were found, yet testing was requested
```

**Comment by suztomo** [comment]
It's better to do one by one.

**Pull request closed without merging**